src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -1,9 +1,35 @@
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
 <head>
-    <title>My team</title>
+    <title style="text-align: center">WEB PRACTICE</title>
 </head>
 <body>
-<h1>Hello, mates!</h1>
+<table style="margin-left:auto;margin-right:auto;">
+    <tr style="font-size: 30px;">
+        <td style="text-align:center; padding: 15px">
+            <a href="/cars" style="color: green; text-decoration: none">CARS</a>
+        </td>
+        <td style="text-align:center; padding: 15px">
+            <a href="/cars/create" style="color: green; text-decoration: none">CREATE CAR</a>
```

Comment:
```suggestion
            <a href="/cars/create" style="color: green; text-decoration: none">CREATE A CAR</a>
```
![image](https://user-images.githubusercontent.com/117590814/214787277-bd6cf302-7e86-4df9-9614-ba6d02e03364.png)
I'm not an english expert :stuck_out_tongue: , but I think this is the proper way